fix(plaxt): move to stable and added docs (#3634)
truecharts/charts: charts/stable/plaxt/docs/installation_notes.md

# Installation Notes

## TRAKT.TV Setup

> - Please create a [app](https://trakt.tv/oauth/applications/new) in trakt.tv.

Reviewer: ```suggestion
- Please create an [app](https://trakt.tv/oauth/applications/new) in trakt.tv.
```

- Set the trakt app name to whatever you like for ex: `plaxt`.

> - Optionally set an icon, you can download this [icon](https://raw.githubusercontent.com/truecharts/charts/9056d3c8f7893e02a070b542cc7985274a05aad2/charts/incubator/plaxt/icon.png).

Reviewer: ```suggestion
- Optionally set an icon, you can download this [icon](https://truecharts.org/img/hotlink-ok/chart-icons/plaxt.png).
```

- Set the redirect uri to for ex (must end with `/authorize`):
- `https://plaxt.mydomain.com/authorize`.
- `https://my_public_ip:8000/authorize`.
- Set the option `/scrobble` to `true`.
- Copy both the Client ID and Secret as you will need both to setup the `plaxt` chart in the next section.

## Plaxt Setup

- Set `TRAKT_ID` with your custom trakt.tv app's client id.
- Set `TRAKT_SECRET` with your trakt.tv app's client secret.

- After launching the chart it will ask you to add your plex username in `Step 1`.
- Click the `authorize` button to authorize your custom app aka `plaxt` with trakt.tv.
- If successful, you will be given a generated webhook to add to plex in `Step 2`.

- Go to plex -> settings -> webhooks -> add the webhook.

> - profit.

Reviewer: ```suggestion
- Profit.
```
